packaging/neo4j-desktop/src/main/java/org/neo4j/desktop/config/osx/DarwinInstallation.java

```diff
@@ -48,25 +49,25 @@ public DarwinInstallation()
                     "echo neo4j-import",
                     "bash"};
 
-            try ( FileWriter fileWriter = new FileWriter( new File( filename ), false ) )
+            try( FileWriter fileWriter = new FileWriter( new File( filename ), false ) )
             {
-                for ( String scriptCommand : scriptCommands )
+                for( String scriptCommand : scriptCommands )
                 {
-                    fileWriter.write( scriptCommand + "\n" );
+                    fileWriter.write( scriptCommand + "\n");
                 }
-            }
 
             String commands[] = { "bash", "-c", "chmod a+x " + filename };
 
             Runtime.getRuntime().exec( commands );
-        }
-        catch( IOException ioe )
-        {
-            System.out.println( "Error writing openNeoTerminal.sh" );
+            }
+            catch( IOException ioe )
+            {
+                System.out.println( "Error writing " + filename );
```

Comment:
The exception messaging to the end user is something I want to work on but I haven't done it yet.